src/pci/pci.h

```diff
@@ -318,19 +318,6 @@ class ProjectorCI : public SelectedCIMethod {
                                                   std::vector<double>& C);
     /// Estimates the projective energy
     double estimate_proj_energy(const det_hashvec& dets, std::vector<double>& C);
-    /// Estimates the variational energy
-    /// @param dets The set of determinants that form the wave function
-    /// @param C The wave function coefficients
-    /// @param tollerance The accuracy of the estimate.  Used to impose |C_I
-    /// C_J| < tollerance
-    double estimate_var_energy(const det_hashvec& dets_hashvec, std::vector<double>& C,
-                               double tollerance = 1.0e-14);
-    /// Estimates the variational energy within a given error
-    /// @param dets The set of determinants that form the wave function
-    /// @param C The wave function coefficients
-    /// @param max_error The accuracy of the estimate. |E_est - E_var|<max_error
-    double estimate_var_energy_within_error(const det_hashvec& dets_hashvec, std::vector<double>& C,
-                                            double max_error = 0.0);
     /// Estimates the variational energy within a given error by sigma vector algorithm
```

Comment:
@tyzhang1993: what does "Estimates the variational energy within a given error" mean here? How is the error defined? I think it would be helpful to explain it.